lib/handleRequest.js

```diff
@@ -57,19 +57,60 @@ function handleRequest (req, res, params, context) {
   return
 }
 
-function jsonBody (request, reply) {
-  var body = ''
-  var req = request.req
-  req.on('error', onError)
+function jsonBody (request, reply, limit) {
+  const contentLength = Number.parseInt(request.headers['content-length'], 10)
```

Comment:
We should not assume Content-Length is present.

Reply:
The short circuit is good. Just wrap it with a check for the header first.